.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docker:
```

Comment:
I suggest adding `if: ${{ secrets.DOCKERHUB_PUSH }}` here so that this workflow does not un-intentionally run on forks unless desired.